helpdesk/views/kb.py

```diff
@@ -46,9 +46,11 @@ def vote(request, item):
     item = get_object_or_404(KBItem, pk=item)
     vote = request.GET.get('vote', None)
     if vote in ('up', 'down'):
-        item.votes += 1
-        if vote == 'up':
-            item.recommendations += 1
-        item.save()
+        if request.user not in item.voted_by: 
+
+            item.votes += 1
+            if vote == 'up':
+                item.recommendations += 1
+            item.save()
```

Comment:
What happens if the user downvotes?

Reply:
Hmm I think it used to have a very similar behavior of recording down votes. Wonder if it was lost in a past refactoring? We'll have to add it back.